data/faq/writing_a_great_proposal.md

```diff
@@ -14,3 +14,23 @@ We want this to be a fun and educating experience for you!
 
 Use the"draft" feature on the GSoC system, submit your draft early, get a
 lot of helpful feedback from us!
+
+
+#### Writing the proposal
+
+Please follow the **application template** in the faq. All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
+
+For writing your proposal we recommend using [Google Docs](https://www.google.com/docs/about/) as this will be supported by the Google submission system.
+
+**In the `code sample` section:** Please list links to your contributions to coala **or any other OpenSource project**. Your contributions may or may not be available on github or related to coala!
```

Comment:
put this kind of info right into the template, we only had this outside because the template wasn't ours